torchinfo/layer_info.py

```diff
@@ -291,6 +291,13 @@ def num_params_to_str(self, reached_max_depth: bool) -> str:
         leftover_params = self.leftover_params()
         return f"{leftover_params:,}" if leftover_params > 0 else "--"
 
+    def per_params_to_str(self, total_params: int) -> str:
+        leftover_params = self.leftover_params()
```

Comment:
This implementation doesn't work - leftover params does not include the number of parameters below this node.

Reply:
Fixed it to match the above function